channels/in.m3u

```diff
@@ -186,8 +186,8 @@ https://live-indiatvnews.akamaized.net/indiatv-origin/liveabr/playlist.m3u8
 http://103.199.161.254/Content/jaihind/Live/Channel(Jaihind)/index.m3u8
 #EXTINF:-1 tvg-id="JanTV.in" tvg-name="Jan TV" tvg-country="IN" tvg-language="Hindi" tvg-logo="" group-title="",Jan TV (360p)
 http://jantvstream.in:1935/edge1/sc1jantv.stream_aac/playlist.m3u8
-#EXTINF:-1 tvg-id="JanamTV.in" tvg-name="Janam TV" tvg-country="IN" tvg-language="Malayalam" tvg-logo="" group-title="Entertainment",Janam TV (240p)
-https://vidcdn.vidgyor.com/janamtv-origin/liveabr/janamtv-origin/live3/playlist.m3u8
```

Comment:
We don't remove an alternate sources if those keep working. A link from @LaneSh4d0w is offering 720p
https://vidcdn.vidgyor.com/janamtv-origin/liveabr/playlist.m3u8

Reply:
Took care of it.

Reply:
This link taking time to load and missing audio after a while for me in VLC.

Reply:
It takes time because it's bumping quality. Purpose of an adaptive playlist.